v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/values/KafkaAuthenticationMethod.java

```diff
@@ -20,7 +20,7 @@ public class KafkaAuthenticationMethod {
   // in @Template.Parameter annotations, and annotations can only have constant attributes.
 
   public static final String NONE = "NONE";
-  public static final String SASL_PLAIN = "SASL_PLAIN";
+  public static final String SASL_PLAIN = "SASL_SSL";
```

Comment:
Is "SASL_SSL" a typo? I haven't seen this constant before.
If it is, please fix it.
If it's not, I suggest keeping the variable name the same (`String SASL_SSL = "SASL_SSL"`).
Otherwise SASL_PLAIN = SASL_SSL and SASL_MECHANISM = PLAIN is very confusing.

Reply:
From the documentation, GMK supports SASL/PLAIN - which is plain username/password encrypted with TLS/SSL. It should be set like this - 
```
sasl.mechanism=PLAIN
# Configure SASL_SSL if TLS/SSL encryption is enabled, otherwise configure SASL_PLAINTEXT
security.protocol=SASL_SSL
```

We were setting SASL_PLAIN which not a valid value. I agree we need a good naming.

Reply:
More info - https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_plain.html